docs/data/toolpad/core/components/app-provider/AppProviderBasic.tsx

```diff
@@ -38,7 +39,11 @@ export default function AppProviderBasic() {
 
   return (
     // preview-start
```

Comment:
Just to note, these comments are not absolutely necessary, the script would likely output the same content automatically in this demo.